geniza/corpus/templates/corpus/document_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load static i18n %}
+
+{% comment %}Translators: Title for document search page{% endcomment %}
+{% translate 'Search Documents' as doctitle%}
+
+{% block title %}{{ doctitle }}{% endblock title %}
+
+{% block main %}
+<div class="container search">
+    <h1>{{ doctitle }}</h1>
+
+    <form>
+    {{ form }}
+    {% for field, errors in form.errors.items %}
+    {# no clean way to get field labels here, so we just collapse the error lists #}
+        {% for error in errors %}
+            <li>{{ error }}</li>
+        {% endfor %}
+    {% endfor %}
+    </form>
+
+    {% comment %}Translators: number of search results{% endcomment %}
+    {% blocktranslate count counter=total %}
+        1 result
+    {% plural %}
+        {{ counter }} total results
+    {% endblocktranslate %}
+
+    <ol>
+    {% for document in documents %}
+        <li class="result">
+
+            <dl class="tags">
+                <dt class="sr-only">{% translate 'Tags' %}</dt>
+                {% for tag in document.tags %}
+                <dd class="tag">#{{ tag }}</dd>
+                {% endfor %}
+            </dl>
+
+            <h2><b>{{ document.type }},</b> {{ document.shelfmark|join:" + " }}</h2>
+            <p>{% translate 'Input date' %} {{ document.input_year|default:'unknown' }}
+            <p>{% translate 'PGP ID' %} {{ document.pgpid }}</p>
+            <p>{{ document.description.0|truncatechars:135 }}</p>
```

Comment:
what's the `.0` do?

Reply:
displays the first element in a list — we're using `*_t` which is multivalued text

Reply:
maybe a dumb question, but why's the description multivalued?

Reply:
I don't think there's a text field that's not multivalued! But I can double check that, it would be nice if it wasn't

Reply:
I checked, all of the dynamic fields for text are multivalued. I guess we could think about adding one ourselves, but going to leave this for now.